test/instrumentation.test.ts

```diff
@@ -85,6 +112,7 @@ describe("bullmq", () => {
   const instrumentation = new BullMQInstrumentation();
   const connection = { host: "localhost" };
   const provider = new NodeTracerProvider();
+  // const tracer = provider.getTracer("test-tracer");
```

Comment:
```suggestion
```

Reply:
Thanks for spotting this! I was on a real pickle trying to get the tracer to work earlier and left this in by mistake.